### ISSUE TITLE
Aggregate Trento (IT) networks or add the ability to open more networks at once

### ISSUE THREAD

**Issue by sanzoghenzo** [bparmentier/OpenBikeSharing#40]
First time user here, sorry if it's already been asked (I couldn't find any info on that).
In the city where I live (Trento, Italy) there is a bike sharing network that works across few towns (Trento, Lavis, San Michele All'adige, Mezzolombardo, Mezzocorona) that are listed as separate networks (e.motion for Trento and Bicincittà for the others).

I think they should be listed as a single network because I can pick/park a bike in any station of these towns.

Regardless, it could be a good thing to be able to select multiple networks, even if they don't talk to each other (i.e. if I'm commuting by bus/train between two towns that have different bike sharing networks but I use both of them locally).

Keep up the good work!

**Comment by peter-alexander** [comment]
I agree on that. There should be checkboxes to select more networks.

**Comment by mirh** [comment]
https://github.com/eskerda/pybikes/blob/master/pybikes/data/bicincitta.json
If you check here (openbikesharing just queries this api) that's expected. 
Every network is a totally different page to 'scan', so they cannot be merged. 
Maybe ping BicinCittà to see if they can merge different comuni, even though I wouldn't be expecting anything from a company that stores and transmits passwords in plaintext. 

If any, adding more networks could be the thing. 
I guess the most difficult task would be figuring out the right UI to present the user in this case.
